CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(funchook C ASM)
+
+include(CheckSymbolExists)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O2")
```

Comment:
Could you remove this line?
When cmake is executed with `-DCMAKE_BUILD_TYPE=Release`, `-O3` is added to CFLAGS.